entities/Order.java

```diff
@@ -0,0 +1,78 @@
+package entities;
+
+import java.util.Date;
+import java.util.List;
+
+public class Order {
```

Comment:
Please create the base entity first and extend from there the common properties such as `id, createdDate and so on ...` Also we don't do hard delete in this, so have that in mind

Reply:
Already, a baseEntity was created; it was extended to product and order entities. Additionally, the readme was updated. Finally, the delete method was removed in the order entity